src/storage.rs

```diff
@@ -225,7 +224,10 @@ impl StorageApi {
         Ok(streaming)
     }
 
-    fn create_rows<M: Message>(table_descriptor: &TableDescriptor, rows: &[M]) -> append_rows_request::Rows {
+    pub fn create_rows<M: Message>(
```

Comment:
I think it would be helpful to add a comment describing how this method works. It seems that it is the caller's responsibility to verify what has actually been sent and to implement a retry mechanism in case there are leftovers (and also even if it can be obvious for most of the people that these leftovers are at the end).

Reply:
Added a comment explaining in detail how the `create_rows` works. This comment together with the example code should demonstrate what typical usage of this and related functions looks like.